docs/dev/release.md

```diff
@@ -22,10 +22,14 @@ This release policy applies to all the aforementioned artifacts.
 
 When performing a full release of all the artifacts, this order should be followed:
 
-1. Runtime
-1. Proto: After executing the release make sure the SDKs are using the latest `proto` version.
-1. SDKs: After runtime and SDK artifacts are published, execute a manual run of the [e2e tests](https://github.com/restatedev/e2e/actions/workflows/e2e.yaml) to check everything works fine.
-1. Documentation and examples
+1. [Runtime](#releasing-the-restate-runtime)
+1. [Proto](https://github.com/restatedev/proto/)
+1. After releasing proto you might need to update the descriptors in the SDKs and examples (`buf.lock` files or `git subtree` updates, refer to the specific release docs of repos)
```

Comment:
Ideally we should automate this, e.g. have a workflow that PRs the update of git subtree.

Reply:
Can we a bit more specific about which repos one needs to go to? I've also been looking at `sdk-java` and `sdk-typescript` and I couldn't find any documentation about how to import the latest proto file definitions. Can we add this if it is indeed missing (useful even if we have the automation)?

Reply:
Yep will do.

Reply:
Done

Reply:
The typescript SDK does not seem to document how it imports the service protocol. It looks as if it is simply copied over.

Reply:
Is it somewhere documented how to update the service protocol in the `restate` repo?